src/polyfill.js

```diff
@@ -10,7 +9,8 @@ import patchFetch from './patch-fetch';
 
   self.AbortController = AbortController;
   self.AbortSignal = AbortSignal;
-
-  patchFetch(self);
+  const {fetch, Request} = abortableFetch(self);
+  self.fetch = fetch;
+  self.Request = Request;
 
 })(typeof self !== 'undefined' ? self : this);
```

Comment:
shouldn't we use `global` here? it's going to be a standard

Reply:
yea, I was thinking maybe we could replace "this" with "global" since it transpiles to "undefined" in ES6 modules.. I just need to check that it works inside web workers on all browsers